pkg/cnb/project_descriptor.go

```diff
@@ -95,10 +121,24 @@ type build struct {
 	Include    []string      `toml:"include"`
 	Exclude    []string      `toml:"exclude"`
 	Buildpacks []buildpack   `toml:"buildpacks"`
+	Group      []buildpack   `toml:"group"`
 	Builder    string        `toml:"builder"`
 	Env        []envVariable `toml:"env"`
 }
 
 type descriptor struct {
 	Build build `toml:"build"`
 }
+
+type descriptorV2 struct {
+	Project project `toml:"_"`
+	IO      ioTable `toml:"io"`
+}
+
+type project struct {
+	SchemaVersion string `toml:"schema-version"`
+}
+
+type ioTable struct {
+	Buildpacks build `toml:"buildpacks"`
```

Comment:
is this missing a level? isn't it `io.buildpacks.build` ?

Reply:
It was changed later in the spec https://github.com/buildpacks/spec/blob/extensions/project-descriptor/0.2/extensions/project-descriptor.md#iobuildpacksgroup-optional